src/main/kotlin/no/nav/familie/klage/ekstern/EksternBehandlingController.kt

```diff
@@ -53,14 +53,40 @@ class EksternBehandlingController(
         val antallTreff = behandlinger.entries.associate { it.key to it.value.size }
         logger.info("Henter klagebehandlingsresultat for eksternFagsakIder=$eksternFagsakIder antallTreff=$antallTreff")
         validerTilgang(behandlinger)
+        return Ressurs.success(behandlinger)
+    }
+
+    @GetMapping("/baks/{fagsystem}")
+    fun hentBehandlingerBAKS(
+        @PathVariable fagsystem: Fagsystem,
+        @RequestParam("eksternFagsakId") eksternFagsakId: String,
+    ): Ressurs<List<KlagebehandlingDto>> {
+        feilHvis(eksternFagsakId.isBlank()) {
+            "Mangler eksternFagsakId i query param"
+        }
+
+        feilHvis(fagsystem !in listOf(Fagsystem.BA, Fagsystem.KS)) {
+            "Ugyldig fagsystem: $fagsystem. Endepunkt støtter kun BA og KS."
+        }
+
+        tilgangService.validerTilgangTilEksternFagsak(eksternFagsakId = eksternFagsakId, fagsystem = fagsystem, event = AuditLoggerEvent.ACCESS)
+
+        logger.info("Henter klagebehandlinger for eksternFagsakId=$eksternFagsakId")
+
+        val behandlinger =
+            behandlingService
+                .finnKlagebehandlingsresultat(
+                    eksternFagsakId = eksternFagsakId,
+                    fagsystem = fagsystem,
+                ).map { it.tilEksternKlagebehandlingDto(behandlingService.hentKlageresultatDto(it.id)) }
 
         return Ressurs.success(behandlinger)
```

Comment:
Nytt endepunkt for BAKS. Vi har aldri mer enn 1 fagsak vi skal hente klagebehandlinger for.